### PR TITLE
JSON-RPC personal service (follows #582)

### PR DESCRIPTION
**Pull request by NikVolf**

### CODE REVIEW

File: rpc/src/v1/impls/personal.rs [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+use v1::traits::Personal;
+use util::keys::store::*;
+use util::{Bytes, Address};
+use std::sync::RwLock;
+
+/// Account management (personal) rpc implementation.
+pub struct PersonalClient {
+	secret_store: Weak<SecretStore>,
+	unlocked_account: Arc<RwLock<Option<Address>>>,
+	unlocked_secret: Arc<RwLock<Option<Bytes>>>,
```
Comment (debris) [inline rpc/src/v1/impls/personal.rs]: isn't secret always `H256`?

File: rpc/src/v1/impls/personal.rs [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+			unlocked_account: Arc::new(RwLock::new(None)),
+			unlocked_secret: Arc::new(RwLock::new(None)),
+		}
+	}
+}
+
+impl Personal for PersonalClient {
+	fn accounts(&self, _: Params) -> Result<Value, Error> {
+		let store = take_weak!(self.secret_store);
+		match store.accounts() {
```
Comment (debris) [inline rpc/src/v1/impls/personal.rs]: minor issue. this function could be less lenient, and return error if there are any input params ;)

**Comment by debris** [comment]
I see that currently it's possible to unlock only 1 account at the same time. Is it the same in geth?

**Comment by NikVolf** [comment]
moved logics to store
now can unlock multiple accounts as in geth

**Pull request merged**